apps/mobile/src/urql-shared/updates/index.js

```diff
@@ -4,73 +4,73 @@ import meCheckAuthQuery from 'graphql/queries/meCheckAuthQuery'
 
 export default {
   Mutation: {
-    clearModerationAction: (result, args, cache, info) => {
-      if (result[info.fieldName].success) {
-        cache.invalidate({ __typename: 'ModerationAction', id: args.moderationActionId })
-      }
-    },
+    // clearModerationAction: (result, args, cache, info) => {
```

Comment:
Some of these updaters can go away for good, some will come back as proper atomic cache updates like createComment, others will still happily invalidate. Right now disabled so they can be re-worked and tested one-by-one outside the scope of this PR.